includes/GameController.hpp

```diff
@@ -84,6 +87,18 @@ class GameController : public Controller
        displayed; by default, the watching period is enabled and the
        boolean is equal to true */
     bool watchingPeriod {true};
+
+    /* this boolean is used to indicate when the current game period is playing
+       and when the player can move over the cells; we do not only use one
+       boolean even if we have only two states because we can have some special
+       cases (visibility bonus for example: the cells are all hidden, so
+       watching, but the player can still move, so playing); when the game
+       starts, the player is watching the level, so the status is playing is
+       false */
+    bool playingPeriod {false};
+
+    /* the color of the player cell */
+    sf::Color playerCellColor;
```

Comment:
the whole color object is used everytime, even if only the apha value is modified